packages/ember-data/lib/system/model/model.js

```diff
@@ -128,6 +128,20 @@ var Model = Ember.Object.extend(Ember.Evented, {
     Ember.deprecate('DS.Model#isDirty has been deprecated please use hasDirtyAttributes instead');
     return this.get('currentState.isDirty');
   }),
+
+  /**
+    @property error
+    @type {Boolean}
+    @deprecated
+  */
+  error: Ember.computed('currentState.isDirty', function(key, error) {
+    if (arguments.length === 2) {
+      this._error = error;
+    }
+    Ember.deprecate('DS.Model#error has been deprecated please use adapterError instead');
```

Comment:
This seems pretty frequent and i imagine users will disable warnings altogether as their apps will get super noisy. We should probably deprecate once.